.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,18 @@
+name: Run Renovate Bot
+
+on:
+  schedule:
+    - cron:  '0/30 * * * *'
```

Comment:
Agree once a day is probably enough. A couple times a day won't hurt anything. It will only create the PRs once. Leaning towards running it a bit faster out of the gate in case we need/want to iterate on the config options more quickly at the beginning.

Reply:
Bumped it down to once an hour